protocol/test/trucoV1/truco-match-factory.ts

```diff
@@ -0,0 +1,58 @@
+import { expect } from 'chai'
```

Comment:
I think this should better be placed at it's own folder at `test/deployer` since the purpose of this folder is to hold tests for core  engine and related logic